app/models/translation.rb

```diff
@@ -39,6 +40,22 @@ def delete_draft!
     return :bad_request
   end
 
+  def translated_pages
+    translated_pages = []
+
+    resource.pages.each do |resource_page|
+      custom_pages.each do |custom_page|
+        if resource_page.id == custom_page.page_id
+          translated_pages.push(custom_page)
+        else
+          translated_pages.push(resource_page)
+        end
+      end
+    end
+
+    translated_pages
```

Comment:
We need to return a list of translated pages.  A translation can have a custom version of the page structure, and if it does, we want to replace the original page with the custom one.  Right now there's no translation happening because the structure of the page hasn't been ironed out yet.  So this is basically a placeholder.